.changeset/plenty-pens-peel.md

```diff
@@ -9,3 +9,4 @@ Update widgets with components from explorer and warp ui
 - Add components: DatetimeField and SelectField
 - New stories: IconList and Fade
 - Add "Icon" suffix for icons that did not have it
+- Update ChainSearchMenu with improvements
```

Comment:
This is fine but FYI you don't have to manually manage your changeset files like this. You can just make a new one for each PR and the tooling with stitch them together when we release